.gitignore

```diff
@@ -7,3 +7,4 @@ webpack.config.js
 coverage
 .nyc_output
 package-lock.json
+*.snap
```

Comment:
Line ending missing here

Reply:
Tests fixed and passing now.
Also fixed the eof error.
Thanks